inc/functions-icons.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * SVG icons related functions and filters.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Seventeen
+ * @since 1.0
+ */
+ 
+/**
+ * Add SVG definitions to the footer.
+ */
+function twentyseventeen_include_svg_icons() {
+	
+	// Define SVG sprite file.
+	$svg_icons = get_template_directory() . '/assets/images/svg-icons.svg';
+	
+	// If it exists, include it.
+	if ( file_exists( $svg_icons ) ) {
+		echo file_get_contents( $svg_icons );
```

Comment:
I personally use `require_once`. For some cases `file_get_contents` would be safer choice as explained in [CSS tricks article](https://css-tricks.com/using-svg/#article-header-id-7).

I'm fine with both.